web/yo/app/scripts/directives/addmodifytherapy.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+angular.module('oncokbApp')
+    .directive('addModifyTherapy', function ($rootScope, DatabaseConnector, _, $q, FirebaseModel, mainUtils, $window) {
+        return {
+            templateUrl: 'views/addModifyTherapy.html',
+            restrict: 'E',
+            controller: function ($scope) {
+                var therapyUuid = [];
+                if ($scope.modifyMode === true) {
+                    clearData();
+                    initTherapy();
+                } else {
+                    clearData();
+                }
+
+                function clearData() {
+                    $scope.therapy = [[]];
+                    therapyUuid = [[]];
+                    $scope.addTherapyError = false;
+                    $scope.noData = true;
+                    $scope.therapyResult = "";
+                }
+
+                function refreshResult() {
```

Comment:
Could you change the name to `reformatTherapy`? `refreshResult` is confused to me. I misread this function is used for refresh data from firebase.